src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128_1.cs

```diff
@@ -692,6 +692,11 @@ private string ToString([StringSyntax(StringSyntaxAttribute.NumericFormat)] stri
         // New Surface Area
         //
 
+        static bool ISimdVector<Vector128<T>, T>.AnyMatches(Vector128<T> vector)
```

Comment:
We did review/approve this (https://github.com/dotnet/runtime/issues/98055#issuecomment-2032720254) and settled on `bool Any(Vector128<T> vector, T value)` and `bool AnyWhereAllBitsSet(Vector<T> vector)`

It would be nice to fix this to follow that. The PR otherwise looks good and should be mergeable.

Reply:
So, the way I understand is we do not need `AnyMatches()` anymore

And `Any`  and `AnyWhereAllBitsSet `would look something like follows

```

static bool ISimdVector<Vector512<T>, T>.AnyWhereAllBitsSet(Vector512<T> vector)
 {
            return (vector.EqualsAny(Vector512<T>.AllBitsSet));
 }
static bool ISimdVector<Vector512<T>, T>.Any(Vector512<T> vector, T value)
{
           return (vector.EqualsAny(Vector512.Create((T)value)));
 }
```

Reply:
Yes, essentially, with the ability for the JIT to recognize these as intrinsic and optimize them more in appropriate scenarios (but that's not necessary for this PR)

Reply:
Done - took out `AnyMatches()` and added `AnyWhereAllBitsSet(`) and `Any()`